
SwipeVelocity: 0.1

OrthogonalDamping: 0.8

OrthVelLim: 0.25

Target_1: [-0.5 , 0.3 , 0.15]

Target_2: [-0.5 , -0.3 , 0.15]
